src/executables/utility.h

```diff
@@ -1691,6 +1693,16 @@ typedef struct _ha_config
 #define FLASHBACK_OLDEST_S          14102
 #define FLASHBACK_OLDEST_L          "oldest"
 
+/* mmon option list */
+#define MMON_MODULE_S         'm'
+#define MMON_MODULE_L         "20123"
```

Comment:
```suggestion
#define MMON_MODULE_L         20123
```
You may want to do this?

Reply:
Exchange to long-term